docs/app/data/components-page.json

```diff
@@ -682,6 +672,37 @@
             ]
         },
         {
+            "title": "Sliders",
```

Comment:
The performance of the sliders was a bit slow when added to the input controls section, however I have moved it back as the performance issue will be improved once we remove the angular 1 components eventually.